domains/yunexiz.json

```diff
@@ -4,8 +4,9 @@
         "email": "yunexiz@aol.com"
     },
     "record": {
-        "A": ["109.71.252.37"], 
-        "TXT": ["v=spf1 include:spf.improvmx.com ~all"],
-        "MX": ["mx1.improvmx.com", "mx2.improvmx.com"]
+    "A": ["185.199.108.153", "185.199.109.153", "185.199.110.153", "185.199.111.153"], 
+    "AAAA": ["2606:50c0:8000::153","2606:50c0:8001::153","2606:50c0:8002::153","2606:50c0:8003::153"],
+    "TXT": ["v=spf1 include:spf.improvmx.com ~all", "_github-pages-challenge-Yunexiz.yunexiz.is-a.dev 273b8f26fd1aacf2cd47be00178179"],
```

Comment:
that's not how you verify the domain with github pages, you should make another file called `_github-pages-challenge-Yunexiz.yunexiz.json` and put the verification TXT record there
see https://www.is-a.dev/docs/github_pages_verification/